pages/profile.css

```diff
@@ -0,0 +1,113 @@
+@import url("https://vercel-ui-preview.netlify.app/component-lib/all-comp.css");
+@import url("https://vercel-ui-preview.netlify.app/component-lib/all-comp.css");
+* {
+	margin: 0;
+	box-sizing: border-box;
+	font-size: 100%;
+}
+html {
+	font-size: 62.5%;
+}
+main {
+	display: grid;
+	height: 100vh;
+	grid-template-rows: auto auto 1fr;
+}
+
+.header {
+	width: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	box-shadow: 0 5px 3px -3px var(--box-shadow);
+}
+
+.header-sub-container {
+	display: flex;
+	align-items: center;
+	gap: 0 2rem;
+}
+.search-container {
+	display: flex;
+	gap: 0;
+	background-color: var(--hover-color);
+}
```

Comment:
you can remove do .header, .header-sub-container, . search-container{ display: flex } since it is repeating.

Reply:
If remove flex from sub-containe,and search-container the layout breaks that why put it.